torch/_inductor/codegen/common.py

```diff
@@ -1397,6 +1397,8 @@ def output(self, name: str) -> str:
         return self._lookup("out_ptr", self.output_buffers, name)
 
     def make_inplace(self, input_name: str, output_name: str) -> None:
+        if input_name in V.graph.unaligned_buffers:
+            V.graph.unaligned_buffers.add(output_name)
```

Comment:
What about buffer reuse ? I suppose with graph inputs we have not had to worry about this as much because we dont reuse them quite as often.

Reply:
Easiest way is to add alignment predicate into the buffer reuse key. Other thoughts?

Reply:
or just not re-use the buffer if it's unaligned, since access should be slower and it is a marginal gain there anyway.